### ISSUE TITLE
Error while running gulp

### ISSUE THREAD

**Issue by younesherlock** [linnovate/mean#1546]
When I run gulp, I get the following error : 

> [00:03:31] [nodemon] watching: _._
> [00:03:31] [nodemon] starting `node --debug server.js`
> debugger listening on port 5858
> 
> util.js:555
>   ctor.prototype = Object.create(superCtor.prototype, {
>                           ^
> TypeError: Object prototype may only be an Object or null
>     at Function.create (native)
>     at Object.exports.inherits (util.js:555:27)
>     at Object.<anonymous> (/home/vendetta/sherlockmean/node_modules/meanio/lib/core_modules/events/events.js:10:6)
>     at Module._compile (module.js:456:26)
>     at Object.Module._extensions..js (module.js:474:10)
>     at Module.load (module.js:356:32)
>     at Function.Module._load (module.js:312:12)
>     at Module.require (module.js:364:17)
>     at require (module.js:380:17)
>     at Object.<anonymous> (/home/vendetta/sherlockmean/node_modules/meanio/lib/core_modules/module/index.js:11:14)
> [00:03:32] [nodemon] app crashed - waiting for file changes before starting...
> [00:03:32] Finished 'less' after 2.23 s
> [00:03:32] Finished 'sass' after 1.92 s
> [00:03:42] csslint: 3 files lint free
> [00:03:42] Finished 'csslint' after 12 s
> ^C^C

**Comment by dsrini-open** [comment]
I am getting the same when I do a $node server command. 
My gulp for some reason throws sass - v for which i started a new issue.

**Comment by younesherlock** [comment]
The fact that this isn't working is really blocking to me... 
I have spent hours trying to find out what might the issue come from, but vainly.
I hope this issue will be considered soon.
Thank you.

**Comment by timelf123** [comment]
Please update to node v5 and report

**Comment by shahfaisal91** [comment]
I was facing the same issue. It's been resolved running below commands:

sudo npm cache clean -f
sudo npm install -g n
sudo n stable

It will update your npm with latest version.
